cmd/govim/flag.go

```diff
@@ -17,12 +18,22 @@ func usage() {
 	fmt.Fprintf(os.Stderr, `
 Usage of govim:
 
-	govim [-tail] /path/to/gopls
+	govim [-tail] [-parent /path/to/uds] gopls ...
+
+See each command's help for more details, e.g.:
+
+   govim gopls -help
```

Comment:
Leftovers from an idea that wasn't implemented?

Reply:
Yes :)